tests/components/rointe/common.py

```diff
@@ -0,0 +1,149 @@
+"""Common data for Rointe tests."""
+from rointesdk.rointe_api import ApiResponse
+
+from homeassistant.components.rointe import (
+    CONF_INSTALLATION,
+    CONF_PASSWORD,
+    CONF_USERNAME,
+)
+from homeassistant.core import HomeAssistant
+
+from tests.common import MockConfigEntry
+
+USERNAME = "rointe@home-assistant.com"
+PASSWORD = "test-password"
+LOCAL_ID = "local-1234"
+INSTALLATION_HOME_ID = "install-home"
+
+MOCK_CONFIG = {
+    CONF_INSTALLATION: INSTALLATION_HOME_ID,
+    CONF_USERNAME: USERNAME,
+    CONF_PASSWORD: PASSWORD,
+}
+
+MOCK_GET_DEVICES_RESPONSE = ApiResponse(True, ["device-1", "device-2"], None)
+
+MOCK_GET_DEVICE_RESPONSES = {
+    "device-1": ApiResponse(
+        True,
```

Comment:
We generally store JSON payloads for tests inside a `fixtures` directory—example: https://github.com/home-assistant/core/tree/dev/tests/components/purpleair/fixtures

Here's an example of how it can be used inside tests: https://github.com/home-assistant/core/blob/a2935654b9237ddc09cf5ea67c8bd967c660ce91/tests/components/purpleair/conftest.py#L76-L81